simplegrid/world.py

```diff
@@ -117,20 +118,29 @@ def step(self):
         self.counts = Counter(creature.__class__.__name__ for creature in self.creatures.values())
 
         game_active = len(self.counts) == 2
+        if not game_active:
+            if len(self.counts) == 0:
```

Comment:
`if not self.counts` ?

Reply:
Nee, want als self.counts == 1 dan is game_active False

Reply:
Uhm, ja. Misschien volg ik het niet. Maar op dit moment is self.counts nul of een. Daar kan je toch direct met je if statement of testen?